sycl/test/sub_group/info.cpp

```diff
@@ -56,14 +55,6 @@ int main() {
                            "global double* c) {*a=*b+*c; }\n");
     kernel Kernel = Prog.get_kernel("kernel_sg");
     uint32_t Res = 0;
-    for (auto r : {range<3>(3, 4, 5), range<3>(1, 1, 1), range<3>(4, 2, 1),
-                   range<3>(32, 3, 4), range<3>(7, 9, 11)}) {
-      Res = Kernel.get_sub_group_info<
-          info::kernel_sub_group::max_sub_group_size>(Device, r);
-      bool Expected = (Res == r.size() || Res == max_sg_num);
-      exit_if_not_equal<bool>(Expected, true,
-                              "max_sub_group_size");
-    }
```

Comment:
Shouldn't we still be testing that the `get_sub_group_info<max_sub_group_size>` query returns meaningful results?

I'd recommend changing this to check that the result of `get_sub_group_info<max_sub_group_size>` exists in the `std::vector<size_t>` returned by `device.get_info<sub_group_sizes>()`.

Reply:
updated